docs/rules/no-restricted-globals.md

```diff
@@ -13,7 +13,7 @@ This rule allows you to specify global variable names that you don't want to use
 
 ## Options
 
-This rule takes a list of strings, where each string is a global to be restricted:
+This rule takes a list of strings.  The first string is the rule setting, such as "off", "warn", or "error".  Each string after the rule setting is a global to be restricted:
```

Comment:
The usage of options was documented here: https://eslint.org/docs/user-guide/configuring#configuring-rules. is there something worth to add in that part?

Reply:
Although it is defined somewhere else, the original sentence is factually incorrect. Furthermore, if a developer doesn't read the page you linked, they will not realize that 'error' is not just a global.

We should be clear in the documentation so that developers won't get confused. What advantage is there in leaving the documentation incorrect?